CONTRIBUTING.md

```diff
@@ -1,23 +1,23 @@
 # Contributing
 
 The general process to contribute code to this repository is as follows: 
-- Fork the repository
 - Create a branch with a detailed name
 - Add your code
-- Commit (with a descriptive message) and push to your fork
-- Create a pull request into the appropriate branch. This is probably the `dev` branch
-- Your PR will be reviewed and merged into dev, where it will be pushed to our development server to validate everything works as expected. Once it's verified that's stable, it will be merged into the master branch.
+- Commit (with a descriptive message) and push to your branch
+- Create a pull request into the appropriate branch. This is probably the `master` branch
+- Ensure that your code passes the CI/CD requirements
+- Your pull request will be reviewed and merged into master, where it will be pushed to our development server to validate everything works as expected
```

Comment:
Where it will be deployed to our production server

Reply:
Good catch